uaclient/defaults.py

```diff
@@ -69,3 +69,4 @@
 WORLD_READABLE_MODE = 0o644
 NOTICES_PERMANENT_DIRECTORY = DEFAULT_DATA_DIR + "/notices/"
 NOTICES_TEMPORARY_DIRECTORY = UAC_RUN_PATH + "notices/"
+USER_LOG_FILE_PATH = "ubuntu-pro/ubuntu-pro"
```

Comment:
I think this should be
```suggestion
USER_LOG_FILE_PATH = "ubuntu-pro/ubuntu-pro.log"
```
and we don't need to add the user name to the file name.

During collect-logs, we should add anonymous tags to the user log files: e.g. "user1.log" "user2.log", etc

Reply:
This might be me overthinking, but I imagine that the root/admin might need to know which user had run which command, thus I thought adding the user will make it easier for them.